src/quest_manager/models.py

```diff
@@ -24,6 +24,7 @@ class Category(IsAPrereqMixin, models.Model):
     """
     title = models.CharField(max_length=50, unique=True)
     icon = models.ImageField(upload_to='icons/', null=True, blank=True)
+    short_description = models.CharField(max_length=500, blank=True, null=True)
```

Comment:
Can you confirm this matches the short description setting for quests? are they both 500 max length?